### DISCUSSION TITLE
这会不会变成百度网盘

### DISCUSSION THREAD

**Discussion by Ann3tt** [share121/inter-knot#729] [其他]
比如说在绳网上传黄油之类的

**Comment by aiTunes-dev** [comment]
应该不会，因为站长不给发，但人多就不行了，管不过来   除非有管理（

**Comment by capacitor1** [comment]
这没办法直接上传大文件。还不得靠百度网盘分享链接。而且不能直接发，会被删

**Comment by Buer-Nahida** [comment]
你可以建一个仓库存，但这违反了github的规定，随时可能被封

**Comment by joke2-54** [comment]
回答：在之前的帖子中，我曾看到关于这方面的链接。

**Comment by SDCOM-0415** [comment]
似乎是可以的...但是GitHub应该会封禁...因为我小号**fork的项目**有问题都被**ban**掉了！
